frontend/src/components/TaskCalendar.jsx

```
@@ -0,0 +1,577 @@
+      chapterId: null,
+      start,
+      end,
+      priority: "Medium",
+      status: "Pending",
+    };
+    setNewTask(newTask);
+  };
+
+  const handleSelectEvent = (event, e) => {
+    // Use the click event coordinates
+    if (e && e.pageX && e.pageY) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)